### ISSUE TITLE
Does not work well with MBLab character

### ISSUE THREAD

**Issue by mysticfall** [xbodya13/ragdoll_tools_doc#5]
I tried to comment out offending lines that causes a problem with 2.9x (#3) and managed to installed the addon.

However, I can't seem to get it to generate a reasonable hitbox setup for the [MBLab](https://github.com/animate1978/MB-Lab) character that I created for a test:

![image](https://user-images.githubusercontent.com/2367032/111021549-832c6500-8410-11eb-9fc4-577d4cb188a9.png)

I'm not sure if it's due to #3, or it's another issue. Either way, it looks like the addon doesn't seem to work for me so I'm afraid I'll have to request a refund unless those issues could be resolved.

**Comment by xbodya13** [comment]
Try different settings  for different bones ,as shown here:
https://youtu.be/lobKcsbQqSY?t=504

**Comment by mysticfall** [comment]
Yes, generating from bones seems to work. But I can't get a usable shapes out of the MBLab mesh, however.

**Comment by xbodya13** [comment]
> But I can't get a usable shapes out of the MBLab mesh, however
Even changing of the  **Threshold** value doesn't help?

**Comment by mysticfall** [comment]
I don't know... it certainly changes the shape but I haven't been able to achieve a usable result yet. Most of the times, the boxes either overlap each other or don't cover the sufficient part of the original mesh.

**Comment by xbodya13** [comment]
If hitboxes will be connected with rigid body constraint, overlapping is ok.
Hitbox shape depends on corresponding vertex group,so you can tweak it to make a better shape.
Also,not every bone needs a hitbox,in case of mblab ,making hitboxes for these bones is a bad idea

![image](https://user-images.githubusercontent.com/14851463/111036396-a9a9ca80-8427-11eb-97be-4da942dd6869.png)
